superchain/configs/sepolia/superchain.toml

```diff
@@ -6,6 +6,7 @@ canyon_time =  1699981200 # Tue 14 Nov 2023 17:00:00 UTC
 delta_time =   1703203200 # Fri 22 Dec 2023 00:00:00 UTC
 ecotone_time = 1708534800 # Wed 21 Feb 2024 17:00:00 UTC
 fjord_time =   1716998400 # Wed 29 May 2024 16:00:00 UTC
+granite_time = 1723478400 # Mon Aug 12 16:00:00 UTC
```

Comment:
The activation time should only be set in the `superchain.toml` and then pulled into all other chain's configurations automatically. We only specify in individual chain configs if we need to _override_ the value from the superchain config.